typings/index.d.ts

```diff
@@ -1001,37 +1017,37 @@ export class Markup {
 
   extra(options?: object): object;
 
-  keyboard(buttons: (Buttons | string)[], options?: object): tt.InlineKeyboardMarkup;
+  keyboard(buttons: (Buttons | string)[], options?: object): Markup;
 
   resize(value?: boolean): Markup;
 
   oneTime(value?: boolean): Markup;
 
-  inlineKeyboard(buttons: CallbackButton[] | CallbackButton[][], options: object): tt.InlineKeyboardMarkup;
+  inlineKeyboard(buttons: CallbackButton[] | CallbackButton[][], options?: object): Markup;
```

Comment:
I'd like to add that inlineKeyboard's buttons also accepts `UrlButton[] | UrlButton[][]`

Reply:
Or maybe even simply `Buttons[] | Buttons[][]`